tests/main/interfaces-autopilot-introspection/task.yaml

```diff
@@ -11,7 +11,6 @@ systems:
     - -ubuntu-14.04-*  # no tests.session
     - -ubuntu-core-*  # no session bus (except for core20+)
     - -amazon-linux-2-*  # no session bus
-    - -centos-7-*  # no session bus
```

Comment:
Does centos 9 support session bus, bfw module, python3 dbus bindings, quota groups, and the newer systemd so that all the below exclusions for centos 7 don't apply to centos 9?

Reply:
the test seems to be working in centos-9, just old versions had those session bus limitations